OSVectorTileAPI/StylingVTS/ExternalFile/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>OS Vector Tile API Demo</title>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script type="text/javascript" src=https://cdnjs.cloudflare.com/ajax/libs/proj4js/2.5.0/proj4-src.js></script>
+		<script type="text/javascript" src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js"></script>
+		<script type="text/javascript" src="ol-mapbox-style/olms.js"></script>
+        <link rel="stylesheet" href="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/css/ol.css" />
+        <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600" rel="stylesheet" />
+        <link href="../../style.css" rel="stylesheet"/>
+    </head>
+    <body>
+        <div id="header">
+            <h1>OS Vector Tile API demonstrator</h1>
+            <p>This sample shows how to display a map using the OS Vector Tile API and OpenLayers</p>
+            <p id="message">To view the map, please enter a valid API key.</p>
+            <form autocomplete="on" onsubmit="setupLayer(); return false;">
+                <label for="keyInput">Project API Key</label><br/>
+                <input id="keyInput" type="text" placeholder="Enter your API Project Key from the OS DataHub" /><button type="submit">Load map</button>
+            </form>
+        </div>
+        <div id="map"></div>
+        <div id="instructions">
+            <p>To connect to your mapping API: </p>
+            <ol>
+                <li>Go to the <a href="https://osdatahub.os.uk" target="_blank" rel="noopener noreferrer">OS DataHub</a>, </li>
+                <li>Create a project, </li>
+                <li>Add the <b>OS Vector Tile API (VTS)</b> API to your project</li>
+                <li>Copy the project API Key, and enter it into the box above.</li>
+            </ol>
+        </div>
+        <script type="text/javascript" src="custom_style.json"></script>
```

Comment:
The custom_style.json file is actually JavaScript, so the file extension is inappropriate. Given that it is javascript, this example is actually the same as the InCode example, but this example happens to have a larger style definition.
We probably could adapt the sample to turn the custom_style.json into a real json file, and then load the file from JavaScript - that way we would be demonstrating what the example is about.

Reply:
Also, Chrome blocks the attempt to load the json file, so the sample doesn't work

Reply:
Good point - example works for me in chrome though.

I agree though that loading it as proper json is the better way forward here.

Reply:
After reading around I've taken it out completely. Since this demo is intended to run locally on a user's machine and browser security appears to be stopping the loading of local files it feels better this way. A NodeJS example at a later date will add the capability, though will be very different to the existing VTS examples.